src/Maker/AdminMaker.php

```diff
@@ -168,7 +168,7 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             );
             $input->setOption('controller', $this->controllerClassBasename);
         }
-        $input->setOption('services', false);
+        $input->setOption('services', null);
```

Comment:
Does this line makes sense ? What happens if you try to pass a --service file?

Reply:
The line was useless indeed, I removed it.